invenio/ext/login/legacy_user.py

```diff
@@ -59,6 +67,20 @@
 }
 
 
+def isUserAdmin(user_info):
```

Comment:
What about passing `user` object directly to both `isUserAdmin` and `isUserSuperAdmin`?